tests/test_cmd_avm.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
```

Comment:
Note: I forgot to update the copyright year and squashed the branching commit. I guess it is fine.